h2o-dist/index.html

```diff
@@ -367,42 +370,56 @@
         readBuildInfoJsonFile();
       }
     </script>
+    <link rel="stylesheet" type="text/css" href="static/css/main.css">
+    <link rel="stylesheet" type="text/css" href="static/css/bleeding-edge.css">
+    <link rel="stylesheet" href="https://use.typekit.net/gbq5zcz.css">
   </head>
-  <body>
+  <body cz-shortcut-listen="true">
 
     <!--[if lt IE 8]>
       <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
     <div id="container">
-      <div id="title">
-        <p class="links">
-          <a href="http://h2o.ai">H<sub>2</sub>O.ai</a> | <a href="https://github.com/h2oai/h2o-dev">Github</a>
-        </p>
-        <h1><span>H<sub>2</sub>O</span></h1>
-        <p class="version">Version SUBST_PROJECT_VERSION</p>
+      <div id="header">
+          <a href="http://h2o.ai/" id="logo">
+            <img src="https://www.h2o.ai/wp-content/themes/h2o2018/templates/dist/images/h2o_logo.svg"/>
```

Comment:
There should definitely not be anything in this page sourced from the marketing website.  Make a copy in ./static